### PR TITLE
chore: use machinomy v1.6.1

### PR DESCRIPTION
**Pull request by michielbdejong**
todo: include fixes corresponding to https://github.com/sharafian/ilp-plugin-ethereum-asym-client/pull/6

### CODE REVIEW

**Comment by michielbdejong** [comment]
Waiting for @sharafian to move this repo to the interledgerjs org.

**Comment by michielbdejong** [comment]
Thanks! Now waiting for a next release of https://www.npmjs.com/package/machinomy, then we can merge this.

File: index.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ class Plugin extends PluginMiniAccounts {
     this._account = opts.account
     this._db = opts.db || 'machinomy_db'
     this._provider = opts.provider || 'http://localhost:8545'
-    this._minimumChannelAmount = opts.minimumChannelAmount || 100
+    this._minimumChannelAmount = opts.minimumChannelAmount || new BigNumber(100)
```
Comment (dappelt) [inline index.js]: What's the expected type of `this._minimumChannelAmount`? Should this be `opts.minimumChannelAmount || '100'`?

Comment (michielbdejong) [reply index.js]: I guess we could change it to `new BigNumber(opts.minimumChannelAmount || 100)`

**Comment by dappelt** [comment]
How did you test the changes?

**Comment by michielbdejong** [comment]
> How did you test the changes?

They are already deployed on amundsen.ilpdemo.org, and I tested them by connecting to that using https://github.com/michielbdejong/ilp-plugin-ethereum-asym-client/blob/498f5980878eb30697a5a80902090806bfe1a9fd/scripts/test-geth.js

**Review by dappelt — APPROVED** [review]

**Pull request merged**